recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-bsp/u-boot/files/0001-Enable-everything-needed-source-command-to-work.patch

```diff
@@ -0,0 +1,52 @@
+From e94f84b9febb4ff4e41f758a58434fb0641b7642 Mon Sep 17 00:00:00 2001
+From: Oleksandr Tyshchenko <oleksandr_tyshchenko@epam.com>
+Date: Fri, 2 Feb 2018 21:42:51 +0200
+Subject: [PATCH] Enable everything needed source command to work
+
+Fix for the cmd_source was taken here:
+https://lists.denx.de/pipermail/u-boot/2015-July/221152.html
```

Comment:
Why didn't you just backport mentioned patch?

Reply:
done